examples/machine-learning/a4-highgpu-8g/a4high-slurm-blueprint-flex.yaml

```diff
@@ -0,0 +1,448 @@
+# Copyright 2025 Google LLC
```

Comment:
I think we can reuse our existing A4 blueprint since we have everything in the same spot.